### DISCUSSION TITLE
make test-integration

### DISCUSSION THREAD

**Discussion by liupenghui-owen** [Kong/kong#10493] [Help]
Hello. I am trying to setup a local development enviornment of Kong. After building and installing Kong from latest source following DEVELOPER.md on Ubuntu 20.04,  running "make test" seems to be OK, but the command" make test-integration" and " make test-plugins" got the error below, How to fix it? Need help and guidance here, thanks a lot!

BTW, there is no target  "make setup-dev-env" in Makefile.

root@owen-virtual-machine:/home/KONG/kong# make test-integration
busted 2.1.1 already installed, skipping
busted-htest 1.0.0 already installed, skipping
luacheck 1.1.0 already installed, skipping
lua-llthreads2 0.1.6 already installed, skipping
http 0.4 already installed, skipping
ldoc 1.4.6 already installed, skipping
bazel-bin/build/kong-dev-venv.sh: line 38: /home/KONG/kong/@bin/busted: no such file or directory.
make: *** [Makefile:122：test-integration] err 127

Also, after running "kong started",  the command "curl -i -X GET --url http://localhost:8001" got nothing, no reponse.

**Comment by fffonion** [comment]
Fix in https://github.com/Kong/kong/pull/10500
